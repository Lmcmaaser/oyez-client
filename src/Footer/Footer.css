/*mobile view*/
footer {
  display: flex;
  flex-direction: row;
  justify-content: center;
  background-color: white;
  padding: 10px;
}

p {
  padding: 0px;
  margin: 0px;
}

a {
  padding: 10px;
}
/*desktop view*/
@media screen and (min-width: 600px) {
  footer {
    display: block;
    width: 120px;
    position: fixed;
    z-index: 1;
    bottom: 0;
    left: 0;
    background-color: white;
    overflow-x: hidden;
    padding-top: 10px;
    margin: 0px;
    /* for bottom full width footer: */
    /* display: block;
    position: fixed;
    left: 0;
    bottom: 0;
    width: 100%;
    padding: 5px;
    text-align: center;
    background-color: lightgrey; */
  }

  p {
    display: inline-block;
    margin: 10px 5px;
  }
}
