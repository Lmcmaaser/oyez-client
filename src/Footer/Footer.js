import React, { Component } from 'react'
import './Footer.css'

export default class Footer extends Component {
  render() {
    return (
      <div className='Footer'>
        <p>Github: <a href="https://github.com/Lmcmaaser" target="_blank"> Lmcmaaser</a></p>
      </div>
    )
  }
}
