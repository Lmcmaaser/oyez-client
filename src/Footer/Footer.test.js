import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

it('renders a .Footer by default', () => {
  const wrapper = shallow(<Footer />);
  expect(toJson(wrapper)).toMatchSnapshot();
});

it('renders without crashing', () => {
  shallow(<Footer />);
});
