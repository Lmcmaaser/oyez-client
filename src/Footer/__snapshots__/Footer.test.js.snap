// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a .Footer by default 1`] = `
<div
  className="Footer"
>
  <div
    className="contacts"
  >
    <a
      href="https://github.com/Lmcmaaser"
      target="_blank"
    >
      <GrGithub />
    </a>
  </div>
  <div
    className="contacts"
  >
    <a
      href="mailto:lmcmaaser@gmail.com"
      target="_blank"
    >
      <GrMailOption />
    </a>
  </div>
  <div
    className="contacts"
  >
    <a
      href="https://www.linkedin.com/in/laurenmaaser"
      target="_blank"
    >
      <GrLinkedinOption />
    </a>
  </div>
</div>
`;
